translation/translator.go

```diff
@@ -145,7 +147,22 @@ func (t *Translator) GetLocale() string {
 }
 
 func (t *Translator) Has(key string, options ...translationcontract.Option) bool {
+	locale := t.GetLocale()
+	// Check if a custom locale is provided in options.
+	if len(options) > 0 && options[0].Locale != "" {
+		locale = options[0].Locale
+	}
+
 	line := t.Get(key, options...)
+
+	// For `{locale}.json` JSON translations, the loaded files will contain the
+	// entire translation tree; therefore, we will check to see if the key
+	// exists in the array of translations.
+	keyValue := getValue(t.loaded[locale]["*"], key)
```

Comment:
Q: Doesn't judge `lang/en/test.json`?

Reply:
well if we couldn't load from en.json will go to next step. Here will will try load from a specific group.
![image](https://github.com/goravel/framework/assets/84431594/d30ac4f7-968b-47fb-8ad5-3b3f0adb2428)

Reply:
Yes, sorry, I ignored the `line := t.Get(key, options...)` line, but if has `line := t.Get(key, options...)`, I think this line is unnecessary, had loaded `t.loaded[locale]["*"]` in the `Get` method.

Reply:
Yeah, You are right. I will remove it